api.swagger.yaml

```diff
@@ -1845,6 +1845,32 @@ paths:
           description: The UUID or request body JSON was malformed.
         '404':
           description: No manual page with that UUID was found.
+  '/manuals/{manual_page_id}/pdf-export':
+    get:
+      description: Get a manual page in HTML for PDF export.
+      tags:
+        - manuals
+      parameters:
+        - name: manual_page_id
+          description: The Invotra UUID of a manual page.
+          in: path
+          required: true
+          schema:
+            type: string
+            format: uuid
+            example: '01234567-89ab-cdef-1234-56789abcdef0'
+      responses:
+        '200':
+          description: Returns a manual page in HTML.
```

Comment:
We should add a response content definition for text/html here, see eg https://swagger.io/docs/specification/adding-examples/#xml-html

Reply:
Schema added.